dispatch_input_playbooks.json

```diff
@@ -0,0 +1,614 @@
+                    "userCode": "    # overwrite parameters\n    parameters = []\n    \n    # Check valid playbook input for repo. Otherwise default to local.\n    if not any(item[0] for item in playbook_input_playbook_repo):\n        phantom.debug(\"No repo provided, defaulting to local\")\n        playbook_repo_list = [\"local\"]\n    else:\n        playbook_repo_list = [item[0] for item in playbook_input_playbook_repo if item[0]]\n    \n    # Control iteration through playbook inputs to match what custom function is expecting\n    for repo in playbook_repo_list:\n        parameters.append({\n            \"name\": None,\n            \"repo\": repo,\n            \"tags\": ', '.join([item[0] for item in playbook_input_playbook_tags]),\n            \"category\": None,\n            \"playbook_type\": \"input\",\n        })\n\n",
+                    "x": 0,
+                    "y": 326
+                },
+                "3": {
+                    "data": {
+                        "advanced": {
+                            "customName": "playbooks decision",
+                            "customNameId": 0,
+                            "description": "Determines if any matching playbooks were found based in the playbook list utility.",
+                            "join": [],
+                            "note": "Determines if any matching playbooks were found based in the playbook list utility."
+                        },
+                        "conditions": [
+                            {
+                                "comparisons": [
+                                    {
+                                        "conditionIndex": 0,
+                                        "op": "!=",
+                                        "param": "find_matching_playbooks:custom_function_result.data.*.full_name",
+                                        "value": ""
+                                    }
+                                ],
+                                "conditionIndex": 0,
+                                "customName": "playbooks found",
+                                "display": "If",
+                                "logic": "and",
+                                "type": "if"
+                            }
+                        ],
+                        "functionId": 1,
+                        "functionName": "playbooks_decision",
+                        "id": "3",
+                        "type": "decision"
+                    },
+                    "errors": {},
+                    "id": "3",
+                    "type": "decision",
+                    "x": 80,
+                    "y": 474
+                }
+            },
+            "notes": "- Must have at least one input style playbook in whichever repo you choose for this utility to work.\n- Playbook will error out if community repo is provided to ensure that users do not accidentally launch community input playbooks that have not been configured."
```

Comment:
Please add a note that artifact scope will be ignored. That can be a bit surprising for someone not used to how the "indicator_collect" function works.

Reply:
Done